src/main/java/org/janelia/saalfeldlab/n5/KeyValueAccess.java

```diff
@@ -251,6 +264,22 @@ default URI uri(final String uriString) throws URISyntaxException {
 	 */
 	public boolean isFile(String normalPath); // TODO: Looks un-used. Remove?
 
+	/**
+	 * Create a {@link ReadData} through which data at the normal key can be read.
+	 * <p>
+	 * Implementations should read lazily if possible. Consumers may call {@link ReadData#materialize()} to force
+	 * a read operation if needed.
+	 * <p>
+	 * If supported by this KeyValueAccess implementation, partial reads are possible by calling slice on the output {@link ReadData}.
+	 *
+	 * @param normalPath is expected to be in normalized form, no further efforts are made to normalize it
+	 * @return a materialized Read data
+	 * @throws N5IOException if an error occurs
+	 */
+	default ReadData createReadData(final String normalPath) throws N5IOException {
+		return ReadData.from(this, normalPath);
```

Comment:
I think only one of the methods `ReadData.from(KVA, String)` and `KVA.createReadData(String)` should exist.

I think you should remove `ReadData.from(KVA, String)`, and also remove the (then unused) `org.janelia.saalfeldlab.n5.readdata.KeyValueAccessReadData` class.